vendor/github.com/openark/golib/sqlutils/sqlutils.go

```diff
@@ -335,20 +332,18 @@ func ExecNoPrepare(db *sql.DB, query string, args ...interface{}) (sql.Result, e
 
 // ExecQuery executes given query using given args on given DB. It will safele prepare, execute and close
 // the statement.
-func execInternal(silent bool, db *sql.DB, query string, args ...interface{}) (sql.Result, error) {
-	var err error
+func execInternal(silent bool, db *sql.DB, query string, args ...interface{}) (res sql.Result,err error) {
 	defer func() {
 		if derr := recover(); derr != nil {
 			err = errors.New(fmt.Sprintf("execInternal unexpected error: %+v", derr))
 		}
 	}()
-
-	stmt, err := db.Prepare(query)
+var stmt *sql.Stmt
```

Comment:
Oh you do have a spacing issue here.

Reply:
Huh. I wonder how this passed CI? CI checks diff vs. `gofmt`-ed code.